### DISCUSSION TITLE
Winget Support Status

### DISCUSSION THREAD

**Discussion by doctordns** [microsoft/winget-cli#3569] [General]
Got asked a quesion by a client: What is the support status of Winget?  Is it now a fully released Microsoft product and what is the support lifecycle? I am looking for a page or pages that set this info out.

**Comment by mdanish-kh** [comment]
Would [SUPPORT.md](https://github.com/microsoft/winget-cli/blob/master/SUPPORT.md) answer your question?

**Comment by doctordns** [comment]
Thanks for the file reference - I had not thought of looking within GitHub for formal support statermens.

We were looking for a formal statement on Microsoft.com. They thought that winget was going to be part of Windows thus the question. 

From the file, it looks like the product is NOT part of Windows; therefore, it is not subject to the same support.
